### PR TITLE
feat: add `simple-swing-app`

### PR DESCRIPTION
**Pull request by magnus-madsen**

### CODE REVIEW

File: examples/swing/simple-swing-app.flix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    let label = new JLabel("Hello World!");
+    frame.getContentPane().add(label);
+
+    frame.pack();
+    frame.setVisible(true)
+}
+
+def main(): Unit \ IO =
+    SwingUtilities.invokeLater(new Runnable {
```
Comment (JonathanStarup) [inline examples/swing/simple-swing-app.flix:19]: why is this part needed?

Comment (magnus-madsen) [reply examples/swing/simple-swing-app.flix:19]: I got it from the example I found. What's the alternative?

Comment (JonathanStarup) [reply examples/swing/simple-swing-app.flix:19]: you can just call createAndShowGUI ?

**Comment by magnus-madsen** [comment]
@JonathanStarup Fixed

File: examples/swing/simple-swing-app.flix [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import java.lang.Runnable
+
+import javax.swing.JLabel
+import javax.swing.JFrame
+import javax.swing.SwingUtilities
```
Comment (mlutze) [inline examples/swing/simple-swing-app.flix:5]: Unused?

**Review by mlutze — APPROVED** [review]
